homeassistant/components/zwave_js/__init__.py

```diff
@@ -134,6 +146,29 @@ def async_on_node_removed(node: ZwaveNode) -> None:
         # note: removal of entity registry is handled by core
         dev_reg.async_remove_device(device.id)
 
+    @callback
+    def async_on_value_notification(notification: ValueNotification) -> None:
+        """Relay stateless value notification events from Z-Wave nodes to hass."""
+        device = dev_reg.async_get_device({get_device_id(client, notification.node)})
+        value = notification.value or notification.data.get("newValue")
```

Comment:
this can be just notification.value once the lib is bumped

Reply:
Can we change this now?

Reply:
done